README.md

```diff
@@ -1 +1,182 @@
-# cache-manager
\ No newline at end of file
+# Cache Manager
+
+[![Build Status](https://travis-ci.org/janis-commerce/cache-manager.svg?branch=JCN-52-memory-manager)](https://travis-ci.org/janis-commerce/cache-manager)
+[![Coverage Status](https://coveralls.io/repos/github/janis-commerce/cache-manager/badge.svg?branch=JCN-52-memory-manager)](https://coveralls.io/github/janis-commerce/cache-manager?branch=JCN-52-memory-manager)
+
+
+The cache-manager is a module for the management of cache, where data is stored in each cache strategy and retrieved first from the one with the highest priority. 
+
+## Installation
+
+
+```js
+npm install @janiscommerce/cache-manager
+```
+
+In order to work, the package needs a configuration file at the root of the application specifying the port and the host it uses for its redis server.
+```js
+path/to/root/config.json
+```
+The expected object in config.json should be:
+```js
+{
+    host: 'localhost',
+    port: 6739
+}
+```
+
+## Usage:
+```js
+const CacheManager = require('@janiscommerce/cache-manager')
+
+// initialize cache manager
+CacheManager.initialize('Client');
+```
+
+### save data
+
+```js
+
+// save in all cache strategies
+CacheManager.save('key', 'subkey', { message: 'hello friend' })
+```
+### fetch data
+```js
+// fetched data in the fastest first strategy and then in the rest
+CacheManager.fetch('key', 'subkey').then(data => {
+    console.log(data) // '{ message: 'hello friend' }
+})
+```
+
+### reset cache
+```js
+// reset all entities in all strategies
+await CacheManager.reset();
+
+// reset only a specific entity in all strategies
+await CacheManager.resetEntity('key');
+```
+
+## API 
+- `initialize('string client')`
+Initialize the cache manager. Receives as a parameter a string to be able to use it as a prefix
+- `save('string key', 'string subkey', 'value')`
```

Comment:
```suggestion
- `save("key", "subkey", value)`
```
Especificar los tipos de objetos en los parametros y los returns de las funciones, Ej:
Save data in memory and redis
Parameters: `key [String]` the key sarasa... `subkey [String]` `value [Any]`?